userdocs/diagnostics/diagnostics.md

```diff
@@ -4,4 +4,84 @@
    @TechnologyRoot
 }
 
-Documentation on diagnostics emitted by the compiler.
\ No newline at end of file
+Documentation on diagnostics emitted by the compiler and settings that control them.
+
+
+## Diagnostics
+
+Swift diagnostics are classified into errors and warnings. Warnings can only be silenced within
```

Comment:
Assuming the omission of notes and remarks is intentional.

Reply:
Remarks partially since they have to be explicitly enabled. Notes I had in originally and am fine to add back, I removed them because you could see them as being just extra information. As above, happy to completely change this if you come up with something better.